ext/storage_sources/s3_store/s3_storage_source.cpp

```diff
@@ -29,22 +29,27 @@
 #include <wiredtiger.h>
 #include <wiredtiger_ext.h>
 
-#include <aws/core/Aws.h>
 #include "s3_connection.h"
+#include "s3_log_system.h"
+#include <aws/core/Aws.h>
+#include <aws/core/utils/memory/stl/AWSString.h>
+#include <aws/core/utils/logging/AWSLogging.h>
 
 #define UNUSED(x) (void)(x)
 
 /* S3 storage source structure. */
 typedef struct {
     WT_STORAGE_SOURCE storageSource; /* Must come first */
     WT_EXTENSION_API *wtApi;         /* Extension API */
+    uint32_t verbose;
```

Comment:
Really sorry to jump in (I know I wasn't assigned as a reviewer), just an interested reader. Just have a small comment, this might be better off declared as a signed integer. I notice that its set to `-3` when the verbose option isn't present .

Reply:
Yes I agree! I missed this, thanks for pointing this out. You are more than welcome on this review 😄 .